common/footer.php

```diff
@@ -6,6 +6,7 @@
 <!-- js files -->
 <script>
     <?php include 'js/login.js'; ?>
```

Comment:
use src tag of script

Reply:
already changed in this commit
https://github.com/gthorat/fly/pull/6/commits/5acd7fc72fc34477fbbec0c47f77d70f0f232d4c